odl/contrib/tensorflow.py

```diff
@@ -0,0 +1,512 @@
+# Copyright 2014-2017 The ODL development group
```

Comment:
I agree with you and I'll also do some other changes, e.g. split this into two different files